excel/beginner/intro-to-functions-and-formulas/introduction-to-functions.md

```diff
@@ -0,0 +1,55 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - workout
+
+type: normal
+
+category: how to
+
+links:
+  - '[Excel Function List](https://support.office.com/en-us/article/excel-functions-alphabetical-b3944572-255d-4efb-bb96-c6d90033e188){documentation}'
+---
+
+# Introduction to functions
+
+---
+## Content
+
+Functions are predefined formulas.
+
+The best part is, there are tons of built-in functions in Excel.
+
+This is why you don't need to create formulas for most things, you can just make use of Excel's built-in ones.
+
+These functions can be used to calculate a sum, an average, min and max values, perform a count, calculate logarithms and much more. A full list of built-in functions will be in the `links` sections.
+
+Just like formulas, functions start with an equal sign but are followed by a "function name" and an argument.

```

Comment:
There might be functions without arguments (like `NOW()` or functions with more than one argument)

```suggestion
Just like formulas, functions start with an equal sign but are followed by a "function name" and zero or more arguments.
```